### PR TITLE
Updated Period length CLI UX

### PR DESCRIPTION
**Pull request by derekpierre**
**Type of PR:**
- [X] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [ ] 2
- [X] 3

**What this does:**
The change of period lengths causes some CLI displayed values to be incorrect eg. length of staking

**Issues fixed/closed:**
Related to #2594 

**Why it's needed:**

**Notes for reviewers:**
> What should reviewers focus on? 
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

File: nucypher/cli/actions/confirm.py [hunk 1]
```diff
@@ -154,7 +159,8 @@ def confirm_staged_grant(emitter, grant_request: Dict, federated: bool) -> None:
     pretty_request['rate'] = f"{pretty_request['rate']} wei/period * {pretty_request['n']} nodes"
 
     expiration = pretty_request['expiration']
-    periods = (expiration - datetime.now()).days
+    periods = calculate_period_duration(future_time=MayaDT(expiration),
+                                        seconds_per_period=seconds_per_period)
```
Comment (KPrasch) [inline nucypher/cli/actions/confirm.py:163]: 👏🏻 - Thank you for maintaining this.  The staking CLI is a backbone.

**Comment by KPrasch** [comment]
May I request we sneak one feature in this PR to match our docs?  I'm not sure we're supporting `--max-gas-price` as part of `nucypher ursula init`, but it will be good to do so.

**Comment by derekpierre** [comment]
> May I request we sneak one feature in this PR to match our docs? I'm not sure we're supporting --max-gas-price as part of nucypher ursula init, but it will be good to do so.

It seems that we are supporting it. Ran the following on `main` this morning:
```bash
$ nucypher ursula init --provider <PROVIDER> --signer <SIGNER> --network ibex --max-gas-price 100
...

$ nucypher ursula config
...
  "gas_strategy": null,
  "max_gas_price": "100",
  "worker_address": <WORKER ADDRESS>,
 ...
 ```
 
Seems to have been implemented via #2445 (commit: https://github.com/nucypher/nucypher/pull/2445/commits/1cf48ede2411bbff6a20d7b7ac3d2c1b716de5dc)

**Review by vzotova — APPROVED** [review]

**Comment by derekpierre** [comment]
re: https://github.com/nucypher/nucypher/pull/2614#issuecomment-808259380

@KPrasch added a commit to #2617 to reinforce that `--max-gas-price` is optional - https://github.com/nucypher/nucypher/pull/2617/commits/9bc9ce5d19bcab341715e4b9fc1e8d9b4a2cca26. The CLI command examples for `nucypher ursula init` make it seem like it's required.

File: nucypher/cli/actions/confirm.py [hunk 2]
```diff
@@ -92,12 +95,14 @@ def confirm_staged_stake(staker_address: str, value: NU, lock_periods: int) -> b
     return True
 
 
-def confirm_large_stake(value: NU = None, lock_periods: int = None) -> bool:
+def confirm_large_and_or_long_stake(value: NU = None, lock_periods: int = None, economics: BaseEconomics = None) -> bool:
     """Interactively confirm a large stake and/or a long stake duration."""
-    if value and (value > NU.from_tokens(150000)):
+    if economics and value and (value > (NU.from_nunits(economics.minimum_allowed_locked) * 10)):  # > 10x min stake
```
Comment (KPrasch) [inline nucypher/cli/actions/confirm.py:100]: Sensible defaults 🥂

File: nucypher/cli/commands/stake.py [hunk 3]
```diff
@@ -32,7 +32,7 @@
 from nucypher.cli.actions.confirm import (
     confirm_enable_restaking,
     confirm_enable_winding_down,
-    confirm_large_stake,
+    confirm_large_and_or_long_stake,
```
Comment (KPrasch) [inline nucypher/cli/commands/stake.py:35]: 👍🏻

File: nucypher/cli/painting/staking.py [hunk 4]
```diff
@@ -101,13 +101,23 @@ def paint_stakes(emitter: StdoutEmitter,
...
 
         if stake.status().is_child(Stake.Status.UNLOCKED) and not paint_unlocked:
             # This stake is unlocked.
             continue
-        rows.append(list(stake.describe().values()))
+
+        stake_description = stake.describe()
+        if is_inactive:
+            # stake is inactive - update display values since they don't make much sense to display
+            stake_description['remaining'] = 'N/A'
```
Comment (KPrasch) [inline nucypher/cli/painting/staking.py:117]: Minor nitpick, Use `'Inactive'` across the board for consistency.

**Review by KPrasch — APPROVED** [review]
Excellent.  I know we'll avoid confusing our stakers with these changes!

**Pull request merged**